src/shared/components/FormComponents/TransferListController/TransferListController.tsx

```diff
@@ -0,0 +1,98 @@
+import { ChangeEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Controller, FieldValues } from 'react-hook-form';
+
+import { Svg, DataTable, DataTableProps } from 'shared/components';
```

Comment:
pls check if this causes circular dependencies

Reply:
fixed